notebook/supermusr-plotting-frames.ipynb

```diff
@@ -0,0 +1,127 @@
+     },
+     "execution_count": 42,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Grab the frame's data\n",
+    "lower_limit = frame_list[frame]\n",
+    "upper_limit = frame_list[frame+1]\n",
+    "frame_data = detector_data[detector_to_plot][lower_limit:upper_limit]\n",
+    "frame_data"
```

Comment:
I think printing the data like this can be removed, it will probably never be looked at and just adds screen clutter.

Reply:
Done :+1: